FrontEnd/src/components/adminPage/detail-view/UserDetail.jsx

```diff
@@ -1,64 +1,117 @@
+    const navigate = useNavigate();
 
-    const handleDeleteClick = () => {
-        setDeleteModalActive(true);
+    const fetcher = url => axios.get(url).then(res => res.data);
+    const { data, error, isValidating: loading } = useSWR(url, fetcher);
+    if (data && !Object.keys(user).length) {
+        setUser(data);
+    }
+
+    const handleSaveChanges = async () => {
+        const response = await axios.put(
+            url,
+            {
+                name: user.name,
+                surname: user.surname,
+                email: user.email,
+                is_active: user.is_active,
+                is_staff: user.is_staff,
+                is_superuser: user.is_superuser
+            },
+        );
+        if (response.status !== 200) {
+            throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+        setUpdateSuccess(true);
+        setTimeout(() => setUpdateSuccess(false), 3000);
+    };
+
+    const handleDeleteUser = async () => {
+        const response = await axios.delete(url);
+        if (response.status !== 204) {
+            throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+        setUser([]);
+        navigate('/customadmin/users');
     };
 
     return (
         <div className={css['user-detail-page']}>
             <DeleteModal
                 active={deleteModalActive}
                 setActive={setDeleteModalActive}
+                onDelete={handleDeleteUser}
             />
             <div className={css['user-details-section']}>
-                <h2>#22 Тарас Панасюк</h2>
-                {FIELDS.map((field) =>(
-                    <div key={field.id} className={css['form-section']}>
-                        <label className={css['form-info__text']}>{field.label}</label>
-                        <input id={field.form_name} type="text" className={css['form-input']}></input>
-                    </div>
-                ))}
-                <Checkbox id="is_active" title="Активний користувач"/>
-
-                <button className={css['save-button']}>Зберегти зміни</button>
+                <ul className={css['log-section']}>
+                    {loading && <li className={css['log']} >Завантаження ...</li>}
+                    {error && <li className={css['log']}>Виникла помилка: {error}</li>}
+                    {updateSuccess && <li className={css['log']}>Користувач успішно оновлений!</li>}
+                </ul>
+                <ul className={css['form-info__user_text']}>
+                    <li>Ім&apos;я: {user.name}</li>
+                    <li>Прізвище: {user.surname}</li>
+                    <li>Email: {user.email}</li>
+                    <li>Активний користувач: {user.is_active ? 'Так' : 'Ні'}</li>
+                    <li>Персонал: {user.is_staff ? 'Так' : 'Ні'}</li>
+                    <li>Компанія: {user.company_name ? 'Так' : 'Ні'}</li>
+                </ul>
```

Comment:
Could you provide a few screenshots with the pr description? We agreed to use ant design component lib to have a consistent design instead we have many custom components.

Reply:
There is no design